src/Sonarr.Api.V3/Queue/QueueResource.cs

```diff
@@ -55,8 +57,9 @@ public static QueueResource ToResource(this NzbDrone.Core.Queue.Queue model, boo
                 Sizeleft = model.Sizeleft,
                 Timeleft = model.Timeleft,
                 EstimatedCompletionTime = model.EstimatedCompletionTime,
-                Status = model.Status,
+                Status = model.Status.FirstCharToLower(),
                 TrackedDownloadStatus = model.TrackedDownloadStatus,
+                TrackedDownloadState = model.TrackedDownloadState,
```

Comment:
wouldn't we need FirstCharToLower here too, and secondly, does this break the api? (clients might be case sensitive)

Reply:
Not in this case because enums are handled automatically, but status is a string.

Just the v3 beta if they are case sensitive, which shouldn't be a problem.